process-engine/skeleton
# Feature/add demo services

## What did you change?

This PR adds services for demo diagrams (chatbot, facelogin) to the process engine demo.

## How can others test the changes?

- `npm i`
- `npm run build` 
- download https://raw.githubusercontent.com/process-engine/example_processes/develop/solutions/Chatbot/bpmn/chatbot.bpmn and https://raw.githubusercontent.com/process-engine/example_processes/develop/solutions/FaceLogin/bpmn/faceLogin.bpmn 
- facelogin requires configuration, see https://github.com/process-engine/example_processes/tree/develop/solutions/FaceLogin
- import the diagrams with bpmn studio and run them

## PR-Checklist

Please check the boxes in this list after submitting your PR:
